src/linagora.esn.calendar/app/app.js

```diff
@@ -47,4 +47,38 @@
     'esn.onscroll',
     'esn.datetime'
   ]);
-})();
+})(angular);
+
+require('esn-frontend-common-libs/src/frontend/js/modules/notification.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user/user-utils.service.js');
```

Comment:
This won't work. When you require an external Angular module, you should require the file where the module is **defined**. In this line, this is "src/frontend/js/modules/user/user.module.js". Please review all other requirements you did to ensure you require the angular file where there is: `angular.module('thename', [...])` and NOT `angular.module('thename')`